packages/account/src/Components/api-token/__tests__/api-token-table.spec.tsx

```diff
@@ -46,25 +54,23 @@ describe('ApiTokenTable', () => {
 
     it('should render ApiTokenTable', () => {
         expectedTexts.push('Last used');
-        render(
-            <ApiTokenContext.Provider value={mock_props}>
-                <ApiTokenTable />
-            </ApiTokenContext.Provider>
-        );
+        renderComponent({});
         expect(screen.getByText('Token 1')).not.toHaveClass('da-api-token__scope-item--name');
         expectedTexts.forEach(text => {
             expect(screen.getByText(text)).toBeInTheDocument();
         });
     });
 
     it('should render in mobile view', () => {
+        const new_store_config: TStores = {
+            ...store,
+            ui: {
+                ...store.ui,
+                is_mobile: true,
+            },
+        };
```

Comment:
```suggestion
        const mock_store = {
            ui: {
                is_mobile: true,
            },
        };
```

It's cleaner to get a new mocked store rather than spreading the old one

Reply:
You mean this yeah? 
```
const mock_store = mockStore({
            ui: {
                is_mobile: true,
            },
 });
```